spec/server/app.spec.js

```diff
@@ -0,0 +1,12 @@
+const expect = require('chai').expect;
+const supertest = require('supertest');
+
+const api = supertest('http://localhost:3005');
+
+describe('User', () => {
+  it('should return 200', (done) => {
+    api.get('/customerReviews')
+    .set('Accept', 'Application/json')
```

Comment:
What does this line do? (just curious)

Reply:
Updated this test with a better description and removed the .set.
It makes a get request to /customerReviews and checks to see if status code 200 is received.